ServerCore/Pages/Responses/Create.cshtml.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Responses
+{
+    public class CreateModel : EventSpecificPageModel
+    {
+        private readonly ServerCore.Models.PuzzleServerContext _context;
+
+        public CreateModel(ServerCore.Models.PuzzleServerContext context)
+        {
+            _context = context;
+        }
+
+        [BindProperty]
+        public Response PuzzleResponse { get; set; }
+
+        public int PuzzleId { get; set; }
+
+        public IActionResult OnGet(int puzzleId, int eventId)
+        {
+            this.PuzzleId = puzzleId;
```

Comment:
Similarly, please remove 'this.' from property accesses

Reply:
I have a PR that'll go in soon (don't want to break everything currently out for review) and it'll set up automatic coding conventions for the code base. It won't apply to existing code, but I'll do a formatting pass as part of that check-in, so don't worry about these for the moment.